### PR TITLE
Updates for JVM Working Dir edge cases with tests

### PR DESCRIPTION
**Pull request by hlhoots**
#build

### CODE REVIEW

**Comment by hlhoots** [comment]
windows build: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_5-6ooFabEeyWaZwSyDrk3A

**Comment by hlhoots** [comment]
Windows build not started, should pickup latest changes to merge 2 tests into 1 case.  Spawning new Personal build as it had already gotten the old code.

#build

File: dev/com.ibm.ws.kernel.boot_fat/fat/src/com/ibm/ws/kernel/boot/ServerStartTest.java [hunk 1]
```diff
@@ -493,6 +493,89 @@ public void test_SWD_ServerJavaDump() throws Exception {
...
+        String javaCoreLocation = null;
+        String serverWorkingDir = null;
+
+        // Cover executing both scripts that utilize this environment variable
+        if (server.getMachine().getOperatingSystem() == OperatingSystem.WINDOWS) {
+            command = "bin" + File.separator + "server.bat";
+            javaCoreLocation = server.getServerRoot().replace("/", "\\");
+            serverWorkingDir = "C:\\";
+        } else { // Linux
+            command = "bin" + File.separator + "server.bat";
```
Comment (jimblye) [inline dev/com.ibm.ws.kernel.boot_fat/fat/src/com/ibm/ws/kernel/boot/ServerStartTest.java]: Don't think you want the .bat here

Comment (hlhoots) [reply dev/com.ibm.ws.kernel.boot_fat/fat/src/com/ibm/ws/kernel/boot/ServerStartTest.java]: fixed this.

**Review by jimblye — APPROVED** [review]

**Comment by hlhoots** [comment]
#build 
Making sure latest changes to run on linux get picked up.

**Review by tbitonti — APPROVED** [review]

**Comment by hlhoots** [comment]
win and personal builds had errors, but nothing related to my changes.  merging.

**Comment by hlhoots** [comment]
#run-libby-bot
